src/demo/pages/PeoplePickerPage/PeoplePickerPage.tsx

```diff
@@ -0,0 +1,75 @@
+import * as React from 'react';
+import {
+  Link
+} from '../../../index';
+import {
+  ExampleCard,
+  PropertiesTableSet,
+  ComponentPage
+} from '../../components/index';
+
+import { PeoplePickerTypesExample } from './examples/PeoplePicker.Types.Example';
+import { getPageRouteFromState } from '../../utilities/pageroute';
+import { AppState } from '../../components/App/AppState';
+import { IComponentDemoPageProps } from '../../components/ComponentPage/IComponentDemoPageProps';
+
+const PeoplePickerTypesExampleCode = require('./examples/PeoplePicker.Types.Example.tsx');
+
+export class PeoplePickerPage extends React.Component<IComponentDemoPageProps, any> {
+  private _url: string;
+
+  constructor() {
+    super();
+    this._url = getPageRouteFromState(AppState, 'Basic components', 'PeoplePicker');
+  }
+
+  public render() {
+    return (
+      <ComponentPage
+        title='PeoplePicker'
+        componentName='PeoplePickerExample'
+        exampleCards={
+          <div>
+            <ExampleCard title='People Pickers' code={PeoplePickerTypesExampleCode}>
+              <PeoplePickerTypesExample />
+            </ExampleCard>
+          </div>
+        }
+        propertiesTables={
+          <PropertiesTableSet componentName='BasePicker' componentPath='components/pickers/' />
+        }
+        overview={
+          <div>
+            <Link target='_blank' href='http://dev.office.com/fabric/components/PeoplePicker'> PeoplePicker </Link>
```

Comment:
I believe this used to say:
"The PeoplePicker is used to select one or more entities, such as people or groups. Entry points for PeoplePickers are typically specialized TextField-like input fields known as a "well", which are used to search for recipients from a list. When a recipient is selected from the list, it is added to the well as a specialized Persona that can be interacted with or removed. Clicking on a Persona from the well should invoke a PersonaCard or open a profile pane for that recipient."